yogstation/code/modules/power/singularity/particle_accelerator/particle_control.dm

```diff
@@ -10,6 +10,8 @@
 
     strength = 4 // Set the new strength to lvl 4
     strength_change() // Update the emitter
+    update_icon()
 
     if(!active)
-        toggle_power()
\ No newline at end of file
+        toggle_power()
+        update_icon()
```

Comment:
Just add it at the end after the if statement once instead of adding it twice.

Reply:
and done